packages/vue/src/components/molecules/SfSelect/SfSelect.vue

```diff
@@ -76,12 +78,13 @@ import SfSelectOption from "./_internal/SfSelectOption.vue";
 import SfChevron from "../../atoms/SfChevron/SfChevron.vue";
 import SfButton from "../../atoms/SfButton/SfButton.vue";
 import SfOverlay from "../../atoms/SfOverlay/SfOverlay.vue";
-import { focus } from "../../../utilities/directives";
+import { focus } from "../../../utilities/directives/";
+import { clickOutside } from "../../../utilities/directives/";
```

Comment:
```suggestion
import { clickOutside } from "../../../utilities/directives";
```
probably you should adjust your imports to my suggestion

Reply:
@AdamPawlinski